index.js

```diff
@@ -4,6 +4,10 @@ import { preprocess } from './lib/preprocess.js'
 import { evaluate } from './lib/evaluate.js'
 import { onContractEvent } from 'on-contract-event'
 import { RoundData } from './lib/round.js'
+import timers from 'node:timers/promises'
+
+// Tweak this value to improve the chances of the data being available
+const PREPROCESS_DELAY = 60_1000
```

Comment:
This value is 601 seconds, did you mean `60_000` instead?

60 seconds seems a bit long delay to me, how about starting with a smaller value like `10_000` or `30_000`?

Reply:
Actually 60s is the lower boundary, it was suggested to go with 1-5 minutes

Reply:
Fixed the number 😅